src/content/blog/solstice-festival.mdx

```diff
@@ -12,43 +12,52 @@ socialImage: '/src/content/blog/_images/solstice-festival/astro-solstice-festiva
+import { Image } from 'astro:assets';
 import Mention from "/src/components/Mention.astro"
-import EventTime from "./_components/EventTime.astro";
+import { YouTube } from '@astro-community/astro-embed-youtube';
+
 
 import poster from "./_images/solstice-festival/launch.png";
 
 **The planets have aligned to bring you the **Astro Solstice Festival** – A week-long celebration of all things Astro.**
 
-Join us from **June 23rd to June 27th, 2025** for a series of exciting events, including live streams, community challenges, and more. Whether you're a seasoned Astro developer or just getting started, there's something for everyone.
+Running from **June 23rd to June 27th, 2025**, the Astro Solstice Festival was a series of exciting events, including live streams, community challenges, and more. Whether you're a seasoned Astro developer or just getting started, there's something for everyone. Don't worry if you missed the live events – you can catch up on all the content right here and on our [YouTube channel](https://www.youtube.com/@AstroBuild).
 
-[See the full schedule below](#schedule) 👇
+[See all events and releases](#schedule) 👇
 
 <Image src={poster} alt="Astro Solstice Festival poster. See below for content" width="768" />
 
 ## Schedule
 
-More details will be added throughout the week, so check back often and follow us on all the socials.
-
 ### Monday, June 23rd
 
 - **[Introducing Astro Mart](/blog/astro-mart/)**: get your festival swag on, with the launch of [Astro Mart](https://shop.astro.build), our new online store for all things Astro. Try the new Summer 2025 collection for festival-ready sunshine vibes.
 
 ### Tuesday, June 24th
 
-- **[Scrimba Town Hall](https://www.youtube.com/watch?v=ea8kp0Z-gYk)** – <EventTime datetime="2025-06-24T16:00:00Z" />: <Mention name="sarah" /> talks about her journey from Scrimba student to Astro core maintainer, what a regular day maintaining Astro looks like, and [Scrimba's new introductory Astro course](https://scrimba.com/intro-to-astro-c00ar0fi5u?via=astro).
-- **[Learning Astro](https://youtube.com/live/0mKNSwi7Wvk?feature=share)** – <EventTime datetime="2025-06-24T18:30:00Z" />: a live panel discussion featuring Astro course creators [Chris Pennington](https://learnastro.dev/?code=ASTRO_PROMO) and [James Q. Quick](https://scrimba.com/intro-to-astro-c00ar0fi5u?via=astro), moderated by <Mention name="sarah" /> talking about the different ways people learn Astro, tips and tricks for learning, how they got involved producing Astro content, and how they see their roles as teachers in the Astro community.
+<YouTube id="https://www.youtube.com/watch?v=ea8kp0Z-gYk" />
+
+- **[Scrimba Town Hall](https://www.youtube.com/watch?v=ea8kp0Z-gYk)**: <Mention name="sarah" /> talks about her journey from Scrimba student to Astro core maintainer, what a regular day maintaining Astro looks like, and [Scrimba's new introductory Astro course](https://scrimba.com/intro-to-astro-c00ar0fi5u?via=astro).
```

Comment:
```suggestion
- **Scrimba Town Hall**: <Mention name="sarah" /> talks about her journey from Scrimba student to Astro core maintainer, what a regular day maintaining Astro looks like, and [Scrimba's new introductory Astro course](https://scrimba.com/intro-to-astro-c00ar0fi5u?via=astro).
```
Up to you whether we still want the videos linked if it's immediately above? And I'd still suggest using the playlist link (`https://www.youtube.com/watch?v=ea8kp0Z-gYk&list=PL8Qn4kutqAEuDiRlQ7TD2lthxTKtrbQTc&index=5`) if we do